test/external/clone.js

```diff
@@ -164,8 +168,8 @@ test('test github repos that use `standard`', t => {
   }), PARALLEL_LIMIT, err => {
     if (err) throw err
     if (argv.write) {
-      fs.writeFileSync(
-        path.join(__dirname, 'test.json'),
+      writeFileSync(
+        join(__dirname, 'test.json'),
```

Comment:
I think that all usages of `join` in this file could be replaced by using URL instead:

```suggestion
        new URL('test.json', import.meta.url),
```

Reply:
ty, this __dirname was the first solution that poped up in search results